frontend/src/tests/components/Commons/CreateCommonsForm.test.js

```diff
@@ -35,7 +35,7 @@ describe(CreateCommonsForm, () => {
       startingBalance: 1000.00,
       cowPrice: 99.95,
       milkPrice: 5.99,
-      startDate: new Date("2021-01-01"),
+      startingDate: new Date("2021-01-01"),
```

Comment:
This line is what makes me think `react-hook-form` passes a `Date` object to `onSubmit`. I thought it would just be the value of the HTML date input.